### PR TITLE
add julia workshop@Beijing post

### PR DESCRIPTION
**Pull request by Roger-luo**
cc: @findmyway @ViralBShah

### CODE REVIEW

**Comment by ViralBShah** [comment]
Is this a replacement for the other PR? Just let me know when things are ready to merge (and make sure to update the dates).

**Comment by Roger-luo** [comment]
@ViralBShah It is a replacement of the other PR. Could you also review the English version? Thanks

**Comment by Roger-luo** [comment]
@findmyway can you review the Chinese version? thanks. I think it will be good to merge if you and @ViralBShah reviews. I'll update the dates after you review this.

File: blog/_posts/2019-08-31-julia-workshop-beijing-zh_cn.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+title:  Julia夏季会议@Beijing & 本地化奖
+author: 罗秀哲，田俊
+---
+
+[JuliaCN](https://github.com/JuliaCN)在8月24日，中国北京举办了自2016以来第五次Julia会议。我们非常感谢Julia社区对本次活动的支持，
+以及[彩云科技](https://caiyunapp.com/), [微软中国](https://microsoft.com/),
+[集智俱乐部](https://swarma.org), [机器之心](https://syncedreview.com/)对本次活动的大力支持。本次活动有100余人注册，实到50余人。
+线上直播在线人数达1600余人。
+
+本次活动的视频以及上传至[JuliaCN@B站](https://www.bilibili.com/video/av65371788)，相关材料已上传到GitHub：[JuliaCN/MeetUpMaterials](https://github.com/JuliaCN/MeetUpMaterials)。
```
Comment (findmyway) [inline blog/_posts/2019-08-31-julia-workshop-beijing-zh_cn.md]: 以及 -> 已经？

**Comment by Roger-luo** [comment]
@ViralBShah is English version good to go? The Chinese version is ready to merge.

**Comment by ViralBShah** [comment]
Sorry I have not been able to review as I have been traveling. Will do soon.

**Comment by ViralBShah** [comment]
The github usernames in the blog should ideally become URLs that link to those individuals' github accounts.

**Pull request merged**